libs/utils/src/tabulation/tally_reports.ts

```diff
@@ -64,14 +69,157 @@ function getAllWriteInRows({
+  manualContestResults,
+}: {
+  contest: CandidateContest;
+  combinedContestResults: Tabulation.CandidateContestResults;
+  scannedContestResults: Tabulation.CandidateContestResults;
+  manualContestResults?: Tabulation.CandidateContestResults;
+}): TallyReportCandidateRow[] {
+  const candidateTalliesDescending = Object.values(
+    combinedContestResults.tallies
+  )
+    .sort((a: Tabulation.CandidateTally, b: Tabulation.CandidateTally) => {
+      return -(a.tally - b.tally); // sort by descending vote tally
+    })
+    .filter((candidateTally) => !isNonCandidateWriteInTally(candidateTally));
+
+  // The least number of votes for someone is winning the race. Notes:
+  // - winner may change as more results are imported or adjudicated
+  // - winner may not be the overall election winner if the report is filtered
+  // - with multiple seats, multiple candidates will be winners
+  const leastNumberVotesForWinner: number =
+    candidateTalliesDescending.at(contest.seats - 1)?.tally ?? 0;
+
+  const writeInCandidateTalliesDescending = candidateTalliesDescending.filter(
+    (candidateTally) => candidateTally.isWriteIn
+  );
+  const significantWriteInCandidates: Tabulation.CandidateTally[] = [];
+
+  while (
+    significantWriteInCandidates.length <
+      writeInCandidateTalliesDescending.length &&
+    getInsignificantWriteInCount({
+      contestResults: combinedContestResults,
+      significantWriteInCandidateIds: significantWriteInCandidates.map(
+        (c) => c.id
+      ),
+    }) >= leastNumberVotesForWinner
+  ) {
+    significantWriteInCandidates.push(
+      assertDefined(
+        writeInCandidateTalliesDescending[significantWriteInCandidates.length]
+      )
+    );
+  }
+
+  const rows: TallyReportCandidateRow[] = [];
+  let hasSomeWriteInRow = false;
+
+  // each significant write-in candidate gets its own row
+  for (const candidate of significantWriteInCandidates) {
+    hasSomeWriteInRow = true;
+    rows.push({
+      ...addWriteInLabelToName(candidate),
+      scannedTally: scannedContestResults.tallies[candidate.id]?.tally ?? 0,
+      manualTally: manualContestResults?.tallies[candidate.id]?.tally ?? 0,
+    });
+  }
+
+  // bucket insignificant write-ins together
+  const significantWriteInCandidateIds = significantWriteInCandidates.map(
+    (c) => c.id
+  );
+  const scannedInsignificantWriteInCount = getInsignificantWriteInCount({
+    contestResults: scannedContestResults,
+    significantWriteInCandidateIds,
+  });
+  const manualInsignificantWriteInCount = manualContestResults
+    ? getInsignificantWriteInCount({
+        contestResults: manualContestResults,
+        significantWriteInCandidateIds,
+      })
+    : 0;
+  if (
+    scannedInsignificantWriteInCount > 0 ||
+    manualInsignificantWriteInCount > 0
+  ) {
+    hasSomeWriteInRow = true;
+    rows.push({
+      id: 'write-in-other',
+      name:
+        significantWriteInCandidateIds.length > 0
+          ? 'Other Write-In'
```

Comment:
Thinking this could read more clearly as `'Other Write-Ins'` - initially thought this line referred to a single write-in candidate when I saw the screenshots. WDYT?

(Similar note for `'Unadjudicated Write-Ins'`)

Reply:
I started by going from the pre-existing pattern, which was singular "Write-In." @carolinemodic also suggested we pluralize it. A possible rationale for the singular is that while there are multiple votes for "Other Write-Ins," each vote is actually for an individual, "Other Write-In". Similarly, in ballot measure contests, we don't count "Nos" or "No's", it's just "No".

Slack thread here: https://votingworks.slack.com/archives/CEL6D3GAD/p1697017412081099.

I don't care much about / think this matters much either way, but feedback has not been strong enough yet to switch away from what seems like me to be a precedent.

Reply:
Ah, gotcha - that makes sense to me